front/pages/ideas/_userId/_ideaId.vue

```diff
@@ -407,33 +492,138 @@ export default {
       graphqlOperation(getIdea, { ideaId: route.params.ideaId })
     )
 
-    let ideaTags = [
-      'web',
-      'illustration',
-      'graphics',
-      'ui',
-      'adobe',
-      'interface'
-    ]
+    let isLiked = await app.$amplifyApi.graphql(
+      graphqlOperation(getIsIdeaLikedByMe, { ideaId: route.params.ideaId })
+    )
+    isLiked = isLiked.data.getIsIdeaLikedByMe.isLiked
```

Comment:
reassigning like this isnt good. You should use desctuctring if u want get that https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment